src/components/DataTable/index.jsx

```diff
@@ -59,6 +78,12 @@ class DataTable extends Component {
     }
   };
 
+  handleFilterClick = () => {
+    if (this.props.onFilterClick) {
+      this.props.onFilterClick();
```

Comment:
Clicking on the filter icon should open a menu item with "Unassigned" as a menu item. Clicking on "Unassigned" should show bugs/issues that are unassigned.

Reply:
Right now it opens the filter form which has a switch with the word "Also display assigned bugs"  since the default is to display the unassigned bugs , do you mean using drop down button with Unassigned as the default selected?

Reply:
Not necessarily. It should be okay having "Unassigned" not selected by default but let the user select it if they can't find unassigned bugs/issues.